tests/CodeFormatterTests.cs

```diff
@@ -114,5 +114,26 @@ public async Task FSharpProjectsDoNotCreateException()
             Assert.Equal(0, formatResult.FilesFormatted);
             Assert.Equal(0, formatResult.FileCount);
         }
+
+       [Fact]
+        public async Task OnlyFormatFilesFromList()
```

Comment:
We need a `NoFilesFormattedWhenNotInList` test which would basically be this test, but with a non-existent file name in the files array

Reply:
Done!